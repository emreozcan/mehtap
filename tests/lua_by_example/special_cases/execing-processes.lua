local success = os.execute('mkdir -p /tmp/example')
if success then
    print('successfully created directory')
end

local handle = io.popen("date --date='@2147483647' --iso-8601=seconds --utc")
if handle == nil then
    return
end
local stdout = handle:read()
success = handle:close()
if success then
    print('output is:', stdout)
else
    print('error when executing command')
end
