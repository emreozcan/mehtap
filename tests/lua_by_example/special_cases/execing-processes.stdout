successfully created directory
output is:	2038-01-19T06:14:07+03:00
